capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -95,6 +96,16 @@ class ClubPage extends Component {
         });
   }
 
+  handleBookChange = async (gbookID) => {
+    this.state.club.gbookID = gbookID;
```

Comment:
You should use setState instead of setting state like this since setState will trigger a react rerender.

Reply:
I'll change that. Only reason I didn't was I was calling setState below anyways, but can't hurt to call in both places.